api/DeviceProximityEvent.json

```diff
@@ -23,7 +23,8 @@
                   "value_to_set": "true"
                 }
               ],
-              "notes": "See <a href='https://bugzil.la/1462308'>bug 1462308</a>."
+              "version_removed": "89",
+              "notes": "See <a href='https://bugzil.la/1462308'>bug 1462308</a>, <a href='https://bugzil.la/1699707'>bug 1699707</a>."
```

Comment:
Same story as before, with the notes. Don't add a new link or drop them entirely. 🚮

Reply:
All notes dropped. They aren't very useful.